### DISCUSSION TITLE
Setup help

### DISCUSSION THREAD

**Discussion by Ryanexley23** [kellerza/sunsynk#92] [General]
Hi 

I am trying to setup this feature in HA but I am quite a noob with HA and I can’t make it work after serval tries. 

The setup instructions mentions only one serial device, would this include a sonoff Zigbee dongle too? 

I say this because once I had started the sunsynk add on the zigbee2mqtt add on would not start and reported errors

If so how would I set it up with both usb devices connected?

**Comment by kellerza** [comment]
Other devices should not be an issue

Multiple addons/programs will not work if they both try to use/open the same USB device

so 1 app/addon to 1 usb device and you should be ok
